content/tutorials/simulation.md

```diff
@@ -0,0 +1,51 @@
+## Create a FrontLine simulation
+
+Now that we've created our artifact on FrontLine, it is time to create a simulation. A simulation is a configuration of a Gatling test on FrontLine.
+
+Let's get back to the main page by clicking on the Gatling icon of the navbar.
+
+![Return to the simulation page](images/click_gatling_icon_edited.png)
+
+Now let's open the creation modal by clicking on the Add button.
+
+<img src="images/add_simulation_button.png" alt="Open the simulation modal" width="900"/>
+
+Enter "my simulation" as the name, choose "Default" as the team and "frontline.sample.BasicSimulation" as the class name. You can then click on the "Next" button.
+
+![General configuration](images/create_simulation_step_1.png)
+
+In the next step, choose the artifact we created earlier and click on the "Next" button.
+
+![Artifact configuration](images/create_simulation_step_2.png)
+
+In this last step, you'll be asked where you want the injection to take place. We'll choose "US east - North Virginia". Once this configuration finished, you can click on the "Save" button.
+
+![Injection configuration](images/create_simulation_step_3.png)
+
+That's it, we created our first simulation on FrontLine !
+
+## Start load testing !
+
+We're now ready to start load testing on FrontLine. To start the simulation we created previously, click on the <i class="fas fa-play"></i> icon.
+
+<img src="images/start_simulation.png" alt="Start the simulation" width="900"/>
+
+You can see the status of the simulation in the "Status" column, and the logs by clicking on the <i class="fas fa-file-alt"></i> button.
+
+<img src="images/simulation_ongoing.png" alt="Simulation status" width="900"/>
+
+You also have access to advanced metrics about your run, even while the simulation is still running. Click on the <i class="fas fa-chart-area"></i> button to see the reports.
+
+<img src="images/reports.png" alt="FrontLine Reports" width="900"/>
+
+You will find here every information you may need to analyze your run. Take some time to interact with the graphs and change tabs.
+
+## Additional resources
+
+You have successfully started load testing, you can read our documentation if you need more information about our product:
+
+FrontLine documentation: https://beta.gatling.io
+
+Documentation about the creation of an artifact: https://beta.gatling.io/docs/user/artifact_gen
```

Comment:
```suggestion
Documentation about the creation of an artifact: {{< ref "docs/user/artifact_gen" >}}
```

Reply:
This reminds me `artifact_gen` is underscore whereas all the pages here are dashed. Should we dash all the pages?

Reply:
i think dash is better